lib/signs/utilities/predictions.ex

```diff
@@ -46,18 +42,19 @@ defmodule Signs.Utilities.Predictions do
     end)
     |> filter_large_red_line_gaps()
     |> Enum.map(fn prediction ->
-      cond do
-        stopped_train?(prediction) ->
-          Content.Message.StoppedTrain.from_prediction(prediction)
-
-        terminal? ->
-          Content.Message.Predictions.terminal(prediction, station_code, zone)
+      special_sign =
```

Comment:
Since this is only used in the `else` branch, should we move it there, to make it clear that this logic doesn't affect the `StoppedTrain` case?